app/models/mixins/custom_attribute_mixin.rb

```diff
@@ -25,7 +25,7 @@ module CustomAttributeMixin
     end
 
     def self.custom_keys
-      CustomAttribute.where(:resource_type => base_class).distinct.pluck(:name).compact
+      CustomAttribute.where(:resource_type => base_class).distinct.pluck(:section, :name).compact
     end
```

Comment:
@lpichler I couldnt find any other place in the code that uses this particular `custom_keys` other than `_custom_details_for`  in `miq_expression.rb` changed below.
So i hope this is resonable.

I didnt go into it too far but i think reporting doesnt take into account that there could be two custom attributes with the same `name` but with different `section` belonging to the same record.